runtime/autoload/man.vim

```diff
@@ -233,13 +237,16 @@ function! s:verify_exists(sect, name) abort
       let path = s:get_path('', a:name)
     endtry
   endtry
+  " The above call to get_path() may have called split, so there may be
+  " a trailing newline or maybe not.  If there is, get rid of it.
+  let path = substitute(path, '\n\+$', '', '')
   " We need to extract the section from the path because sometimes
   " the actual section of the manpage is more specific than the section
   " we provided to `man`. Try ':Man 3 App::CLI'.
   " Also on linux, it seems that the name is case insensitive. So if one does
   " ':Man PRIntf', we still want the name of the buffer to be 'printf' or
   " whatever the correct capitilization is.
-  let path = path[:len(path)-2]
+  let path = path[:len(path) - 1]
```

Comment:
This needs to be removed. It was supposed to trim the trailing newline but that should be done in `s:get_path`.